### ISSUE TITLE
phishy content rendering

### ISSUE THREAD

**Issue by noumantahir** [ecency/ecency-mobile#2295]
If I understand correctly the purpose of phishy is to render highlighted text inline with leading and trailing content.

Phishy class content rendering is not working as expected as it is rendered as a block content instead of inline content...

Perhaps we can change that using dom tempering

![Screenshot 2022-05-11 at 3 29 00 PM](https://user-images.githubusercontent.com/6298342/167829787-32b19a47-0125-4cce-b85b-b0c6a7ad5669.png)

**Comment by feruzm** [comment]
Yes that would be really great we should just mark the text in a red color or somehow highlight it and that’s it
